### PR TITLE
Chips (tags) on VideoCards

### PR DESCRIPTION
**Pull request by mwilkins74**
Starting to poke at this:

Screen Shot 2024-02-02 at 10 14 01 AM

### CODE REVIEW

**Comment by mwilkins74** [comment]
Colors!!!
Screen Shot 2024-02-02 at 2 01 42 PM

**Comment by mwilkins74** [comment]
Screen Shot 2024-02-02 at 2 45 59 PM

File: src/common/utils/enumColorUtils.ts
Comment (ChuckBTaylor) [inline src/common/utils/enumColorUtils.ts:1]: 💭 Anytime I see hard coded colors, I wonder if there's a way we can access them via the theme.

**Review by ChuckBTaylor — APPROVED** [review]

File: src/common/utils/enumColorUtils.ts
Comment (mwilkins74) [reply src/common/utils/enumColorUtils.ts:1]: Excellent point!
Something we can figure out in the future.

**Pull request merged**